docs/admin/kube-apiserver.md

```diff
@@ -109,6 +109,7 @@ kube-apiserver
       --service-node-port-range=: A port range to reserve for services with NodePort visibility.  Example: '30000-32767'.  Inclusive at both ends of the range.
       --ssh-keyfile="": If non-empty, use secure SSH proxy to the nodes, using this user keyfile
       --ssh-user="": If non-empty, use secure SSH proxy to the nodes, using this user name
+      --storage-media-type="application/json": The media type to use to store objects in storage. Defaults to application/json. Some resources may override this value.
```

Comment:
i.e., change to `--storage-media-type-default`. Introduce the override list when it becomes necessary.